src/ui/public/collapsible_sidebar/collapsible_sidebar.js

```diff
@@ -23,6 +32,17 @@ uiModules
 
       $collapser.on('click', function () {
         $elem.toggleClass('closed');
```

Comment:
If you're moving to a add/remove class instead of toggle class, why not move this down as well?